src/main/java/redis/clients/jedis/activeactive/CircuitBreakerCommandExecutor.java

```diff
@@ -0,0 +1,61 @@
+package redis.clients.jedis.activeactive;
```

Comment:
how about redis.clients.jedis.aa. IMHO activeactive is too long

Reply:
Also - can we add some docstring text forthe package purpose now? It's time to start with this PR.

Reply:
IMO, `aa` is way too short.

Reply:
> Also - can we add some docstring text forthe package purpose now? It's time to start with this PR.

Added.